content/webapp/pages/article.tsx

```diff
@@ -225,7 +228,14 @@ const ArticlePage: FC<Props> = ({ article, jsonLd }) => {
                   </Space>
                 </Fragment>
               ))}
-
+            {readingTime && article.readingTime ? (
+              <Fragment>
```

Comment:
I'd prefer `<>` to `<Fragment>` (it's just an alias, but short is best!).

I think the only time you still need `<Fragment>` is if you need to add a `key` attribute to it.

Reply:
ah very good point David, thanks.